Lib/http/server.py

```diff
@@ -131,7 +132,23 @@
 
 class HTTPServer(socketserver.TCPServer):
 
-    allow_reuse_address = 1    # Seems to make sense in testing environment
+    allow_reuse_address = True    # Seems to make sense in testing environment
+
+    def __init__(self, server_address, RequestHandlerClass,
+            bind_and_activate=True, *, tls=None):
+        if tls is None:
+            self.tls_cert = self.tls_key = None
+        else:
+            self.tls_cert, self.tls_key = tls
+        super().__init__(server_address, RequestHandlerClass, bind_and_activate)
+
+    def server_activate(self):
+        """Wrap the socket in SSLSocket if TLS is enabled"""
+        super().server_activate()
+        if self.tls_cert and self.tls_key:
+            context = ssl.SSLContext(ssl.PROTOCOL_TLS_SERVER)
```

Comment:
use ``ssl.create_default_context``.

Reply:
The default SSL context requires a TLS 1.3 certificate so it doesn't work with `ssl_cert.pem`:

```
ssl.SSLError: [SSL: TLSV13_ALERT_CERTIFICATE_REQUIRED] tlsv13 alert certificate required
```

Is this something that we want to force for a development server?

Reply:
There is no such thing as a *TLS 1.3* certificate. This is a TLS alert with message ``certificate required``. Looks like the certificate does not have correct extensions to handle TLS 1.3.

You cannot use any file from ``test`` directory any way. The directory is often not installed or shipped to reduce disk space usage in containers or on small systems.